draft-ietf-wimse-s2s-protocol.md

```diff
@@ -616,8 +616,8 @@ In order to reduce the risk of theft and replay the PoP should have a limited sc
 
 * Binding to a Timestamp or Nonce
 
-A proof of possession should include information that can be used to uniquely identify it such as a unique timestamp or nonce.  This can be used by the receiver to perform basic replay protection against tokens it has already seen. Depending upon the design of the system it may be difficult to synchronize the replay cache across all token validators. In this case, if the PoP is not sufficiently scoped it may be usable with another workload.
-While a fresh nonce could be included in the PoP, a mechanism for distributing a fresh challenge nonce from the validator to provide single use properties of a PoP is outside the scope of this specification.
+A proof of possession includes the `jti` claim that uniquely identifies it, within the scope of a particular sender.
+This SHOULD be used by the receiver to perform basic replay protection against tokens it has already seen. Depending upon the design of the system it may be difficult to synchronize the replay cache across all token validators. In this case, if the PoP is not sufficiently scoped it may be usable with another workload.
```

Comment:
Are there cases where pop replay is allowed or a 'feature'?

Reply:
No, plus reconsider the last sentence: "not sufficiently scoped".